tests/fuzzers/ucl_add_string_fuzzer.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+#include <errno.h>
+#include <unistd.h>
+#include "ucl.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+	char *ss = (char*)malloc(size+1);
+	memcpy(ss, data, size);
+	ss[size] = '\0';
```

Comment:
Why do you use zero terminated string here? ucl_parser_add_string accepts length as an argument so that will allow to fuzz test inputs with `\0` which are proven to be one of the most dangerous ones.

Reply:
You are completely right here. I have updated the fuzzer.